telegram/rpc.go

```diff
@@ -104,6 +107,12 @@ func (c *Client) rpcDoRequest(ctx context.Context, req request) error {
 		return xerrors.Errorf("write: %w", err)
 	}
 
+	ackCtx, ackClose := context.WithCancel(c.ctx)
+	defer ackClose()
+
+	// resend request until we receive ack or response for it
```

Comment:
>  until we receive ack or response for it

Is is possible to receive both? Looks like server can first ACK and then return responce.

Also please use go-style comments like
```go
// Resent request until we recieve ack.
```
(i.e. just sentence with starts with uppercase letter and ends with dot)